django/contrib/sessions/backends/base.py

```diff
@@ -36,6 +44,9 @@ class UpdateError(Exception):
     pass
 
 
+# RemovedInDjango40Warning: SessionBase is left untouched
+# for backward compatibility purposes, but should be merged
+# into HashingSessionBase.
```

Comment:
If we really want to deprecate `SessionBase` we need to add a deprecation warning etc., see [docs](https://docs.djangoproject.com/en/3.1/internals/contributing/writing-code/submitting-patches/#deprecating-a-feature).